ufficio_soci/forms.py

```diff
@@ -412,13 +412,23 @@ def clean_data_versamento(self):
 
 class ModuloFiltraEmissioneTesserini(forms.Form):
 
+    def __init__(self, *args, **kwargs):
+        sedi = kwargs.pop("sedi")  # passo le sedi permesse
+        sedi_sottosedi = [sede.esplora() for sede in sedi]
+        id_sede = [scelte.id for scelte in sedi_sottosedi[0]]
+        nomi_sede = [scelte.nome_completo for scelte in sedi_sottosedi[0]]
+        scelte_sede = list(zip(id_sede, nomi_sede))
+        super(ModuloFiltraEmissioneTesserini, self).__init__(*args, **kwargs)
+        self.fields['comitato'] = forms.MultipleChoiceField(choices=scelte_sede, initial=id_sede)
+
     stato_richiesta = forms.MultipleChoiceField(choices=Tesserino.STATO_RICHIESTA)
     tipo_richiesta = forms.MultipleChoiceField(choices=Tesserino.TIPO_RICHIESTA, initial=(Tesserino.RILASCIO,
                                                                                           Tesserino.RINNOVO,
                                                                                           Tesserino.DUPLICATO))
     stato_emissione = forms.MultipleChoiceField(choices=Tesserino.STATO_EMISSIONE, initial=(("", Tesserino.STAMPATO,
                                                                                              Tesserino.SPEDITO_CASA,
                                                                                              Tesserino.SPEDITO_SEDE)),)
+    comitato = forms.MultipleChoiceField()
```

Comment:
- `comitato` (che viene anche mostrato sulla pagina) e' singolare, mentre il campo e' a scelta multipla. Inoltre, Sede != Comitato (nella selezione, potresti avere delle sedi territoriali, che non sono Comitati CRI). Suggerisco `sedi`.
- Non usare un `MultipleChoiceField()`. Devi popolarlo manualmente, facendo ogni sorta di stregoneria e conversione tra liste di ID e oggetti nel frattempo. Inoltre, la responsabilita' della validazione ricade su di te! 
  Credo che tu stia cercando [`forms.ModelMultipleChoiceField`](https://docs.djangoproject.com/en/2.0/ref/forms/fields/#modelmultiplechoicefield) a cui puoi fornire direttamente un QuerySet, e fa tutta questa stregoneria per te. Puoi dargli in pasto direttamente un QuerySet, come ad esempio:
  ```python
  assert isinstance(sedi, SedeQuerySet)
  sedi = sedi.espandi(pubblici=True)
  assert isinstance(sedi, SedeQuerySet)
  ```
  (si, puoi chiamare `.espandi` direttamente su un QuerySet di sedi, e ti ritorna un altro QuerySet!)

Reply:
Potresti comunque voler mostrare solamente i comitati Regionali, Provinciali e Locali. Puoi farlo con `SedeQuerySet.comitati()`, ad es.,
```python
sedi = sedi.espandi(pubblici=True).comitati()
```
Se dovessimo avere un elenco di tutte le unita' territoriali, avremmo un elenco infinito! Inoltre probabilmente al Comitato Regionale non importa comunque la distinzione in sedi territoriali. Corretto, @luca-dex?


Inoltre, ricorda che di default dovrebbe essere tutto selezionato.

Reply:
Confermo. il livello da utilizzare, in questo caso, è quello del comitato locale. Tenendo sempre in considerazione le particolarità (area metropolitana di roma capitale, ecc). Non è necessario frammentare a livello di unità territoriale se utilizziamo questo sistema. È comumque importante cosiderare che abbiamo un sistema di selezione dall'albero dei comitato già esistente, perchè non è stato usato quello come pre-filtro a questa pagana, come fatto in altre componenti di GAIA?